metron-deployment/roles/ambari_config/vars/single_node_vm.yml

```diff
@@ -102,7 +102,7 @@ configurations:
 
 required_configurations:
   - metron-env:
-      storm_rest_addr: "{{ groups.ambari_slave[0] }}:8744"
+      storm_rest_addr: "http://{{ groups.ambari_slave[0] }}:8744"
```

Comment:
It probably shouldn't be.  I did that before the backwards compatibility commit.  It'll be assumed at the code level if no protocol is there anyway, so do we have a preference on explicit vs. implicit?

It seems more like I need documentation around changing that parameter when storm https is enabled (both non-Ambari and Ambari need it).